deploy/template.yaml

```diff
@@ -171,6 +171,7 @@ Mappings:
       UADISABLED: "false"
       SUPPORTCHANGEMFA: "1"
       ACCESSIBILITYSTATEMENTURL: "https://signin.account.gov.uk/accessibility-statement"
+      LANGUAGETOGGLE: 1
```

Comment:
I didn't update this, I think it might have been Andrew when he was helping deploy and test my branch. But I have been informed that environment flags are team's responsibility to manage as we're not the codeowners for those repos and we need to be consistent with how we work with each team